Sources/PostgresNIO/New/Data/RawRepresentable+PSQLCodable.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 13.01.21.
+//
+
+extension PSQLCodable where Self: RawRepresentable, RawValue: PSQLCodable {
```

Comment:
Ditch Xcode template headers.
```suggestion
extension PSQLCodable where Self: RawRepresentable, RawValue: PSQLCodable {
```